pkg/operator/controller/gatewayclass/servicemeshcontrolplane.go

```diff
@@ -99,17 +95,17 @@ func desiredServiceMeshControlPlane(name types.NamespacedName, ownerRef metav1.O
 			},
 			Gateways: &maistrav2.GatewaysConfig{
 				ClusterIngress: &maistrav2.ClusterIngressGatewayConfig{
-					IngressEnabled: &t,
+					IngressEnabled: &f,
 					IngressGatewayConfig: maistrav2.IngressGatewayConfig{
 						GatewayConfig: maistrav2.GatewayConfig{
 							Enablement: maistrav2.Enablement{
-								Enabled: &t,
-							},
-							Service: maistrav2.GatewayServiceConfig{
-								ServiceSpec: corev1.ServiceSpec{
-									Type: corev1.ServiceTypeLoadBalancer,
-								},
+								Enabled: &f,
 							},
+							//Service: maistrav2.GatewayServiceConfig{
+							//	ServiceSpec: corev1.ServiceSpec{
+							//		Type: corev1.ServiceTypeLoadBalancer,
+							//	},
+							//},
```

Comment:
I am very much in favor of getting rid of the istio-ingressgateway service.  Have we recorded the decision to eliminate it anywhere?  Perhaps a revision to the existing EP, or a follow-up EP?